package.json

```diff
@@ -10,11 +10,15 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "@auth/mongodb-adapter": "^1.0.0",
     "@next/font": "^13.4.19",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "mongodb": "^5.8.1",
     "mongoose": "^7.5.0",
     "next": "13.4.8",
+    "next-auth": "^4.23.1",
+    "nodemailer": "^6.9.5",
```

Comment:
You're currently not using `nodemailer`. Uninstall it for now. You can always add it back in when you're ready to implement it.